torch/package/_package_unpickler.py

```diff
@@ -23,4 +30,12 @@ def find_class(self, module, name):
             elif module in _compat_pickle.IMPORT_MAPPING:
                 module = _compat_pickle.IMPORT_MAPPING[module]
         mod = self._importer.import_module(module)
+
+        if hasattr(self._importer, "selective_extern_packages"):
```

Comment:
What's going on here? This should be handled internally by `self._importer.import_module`.